src/test/java/core/basesyntax/strategy/impl/OperationHandlerSupplyImplTest.java

```diff
@@ -0,0 +1,56 @@
+package core.basesyntax.strategy.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.dao.FruitDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.db.impl.StorageImpl;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.strategy.OperationHandler;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class OperationHandlerSupplyImplTest {
+    private Storage testStorage;
+    private Fruit apple;
+    private FruitDao testFruitDao;
+    private OperationHandler operationHandlerSupply;
+
+    @Before
+    public void setUp() {
+        apple = new Fruit("apple");
+        testStorage = new StorageImpl();
+        testFruitDao = new FruitDaoImpl();
+        operationHandlerSupply = new OperationHandlerSupplyImpl();
```

Comment:
looks like this fields we could initialise in a method with `BeforeClass` annotation
also we shouldn't use `FruitDaoImpl` class in these tests
<img width="1005" alt="image" src="https://user-images.githubusercontent.com/58032907/200304005-cbac505b-eae9-4ace-8653-ea75f4d398fd.png">